allennlp/modules/stacked_bidirectional_lstm.py

```diff
@@ -0,0 +1,116 @@
+from typing import Optional, Tuple
+import torch
+from torch.nn.utils.rnn import PackedSequence, pack_padded_sequence, pad_packed_sequence
+from allennlp.modules.augmented_lstm import AugmentedLstm
+from allennlp.common.checks import ConfigurationError
+
+
+class StackedBidirectionalLstm(torch.nn.Module):
+    """
+    A standard stacked Bidirectional LSTM where the LSTM layers
+    are concatenated between each layer.
+
+    Parameters
+    ----------
+    input_size : int, required
+        The dimension of the inputs to the LSTM.
+    hidden_size : int, required
+        The dimension of the outputs of the LSTM.
+    num_layers : int, required
+        The number of stacked Bidirectional LSTMs to use.
+    recurrent_dropout_probability: float, optional (default = 0.0)
+        The dropout probability to be used in a dropout scheme as stated in
+        `A Theoretically Grounded Application of Dropout in Recurrent Neural Networks
+        <https://arxiv.org/abs/1512.05287>`_ .
+
+    Returns
+    -------
+    output_accumulator : PackedSequence
```

Comment:
This doesn't match what you have in `forward`.  Do you even need this here at all?